timeline-chart/src/time-graph-container.ts

```diff
@@ -59,13 +58,13 @@ export class TimeGraphContainer {
         this.layers = [];
 
         const background = new TimeGraphRectangle({
-            opacity: 0,
```

Comment:
THIS WAS THE PROBLEM! transparent according to PIXI5 is incorporeal. So according to PIXI 5 I can fly through glass.